### PR TITLE
fix(components): slider change event (VIV-985)

### PR DESCRIPTION
**Pull request by olaf-k**

### CODE REVIEW

File: libs/components/src/lib/slider/slider.ts [hunk 1]
```diff
@@ -24,22 +25,20 @@ export class Slider extends FastSlider {
 	 */
 	override valueChanged(previous: string, next: string): void {
 		if (this.$fastController.isConnected) {
+			// min/max constraints backported from v3
```
Comment (YonatanKra) [inline libs/components/src/lib/slider/slider.ts:28]: I don't understand this comment. 
Can you elaborate please?

**Review by YonatanKra — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/slider/slider.ts (hunk 1 above)
Comment (olaf-k) [reply libs/components/src/lib/slider/slider.ts:28]: the PR fix (mix/max constraints) is backported from Foundation v3 (alpha).

**Review by YonatanKra — COMMENTED** [review]
Mind removing the comments?
They mean nothing to most people.
We have tests that will fail if we upgrade to v3, right?

**Comment by olaf-k** [comment]
> Mind removing the comments? They mean nothing to most people.

I actually do? they are useful to explain why the code is here and why it looks like that. that is a strange request.

> We have tests that will fail if we upgrade to v3, right?

we don't have tests designed to fail when you upgrade Fast's version, no, why would we?

**Comment by YonatanKra** [comment]
> > Mind removing the comments? They mean nothing to most people.
> 
> I actually do? they are useful to explain why the code is here and why it looks like that. that is a strange request.
> 
> > We have tests that will fail if we upgrade to v3, right?
> 
> we don't have tests designed to fail when you upgrade Fast's version, no, why would we?

According to your comments, if we upgrade FAST to v3, the tests are supposed to fail.
Am I correct?

**Comment by olaf-k** [comment]
> According to your comments, if we upgrade FAST to v3, the tests are supposed to fail. Am I correct?

the comments are to remind me/whoever will maintain that:
1) the min/max computation—lacking in the v2 Fast component which we inherit from—is pretty much a copy/paste of the v3. it will not fail upon upgrade but can then be removed as duplicate.
2) the rest of the logic (setThumbPositionForOrientation + change event) is—by opposition to 1—correctly done in the v2 parent class.

**Comment by YonatanKra** [comment]
> > According to your comments, if we upgrade FAST to v3, the tests are supposed to fail. Am I correct?
> 
> the comments are to remind me/whoever will maintain that:
> 
> 1. the min/max computation—lacking in the v2 Fast component which we inherit from—is pretty much a copy/paste of the v3. it will not fail upon upgrade but can then be removed as duplicate.
> 2. the rest of the logic (setThumbPositionForOrientation + change event) is—by opposition to 1—correctly done in the v2 parent class.

@olaf-k 
Anyway - if there's a test covering the `valueChanged` method, I think we can remove these comments. The fact that I (a possible future maintainer) asked what they mean makes the comments rather useless.

**Comment by olaf-k** [comment]
> @olaf-k Anyway - if there's a test covering the `valueChanged` method, I think we can remove these comments. The fact that I (a possible future maintainer) asked what they mean makes the comments rather useless.

you not understanding something doesn't make it useless Yonatan.

**Comment by YonatanKra** [comment]
> > @olaf-k Anyway - if there's a test covering the `valueChanged` method, I think we can remove these comments. The fact that I (a possible future maintainer) asked what they mean makes the comments rather useless.
> 
> you not understanding something doesn't make it useless Yonatan.

I agree. It doesn't make it that way, it hints on it. Need to make some more experiments :)
Can we ditch the comments and merge this?

**Pull request merged**